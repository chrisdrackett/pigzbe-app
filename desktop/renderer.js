import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
