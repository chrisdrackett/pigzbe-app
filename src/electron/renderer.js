import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
