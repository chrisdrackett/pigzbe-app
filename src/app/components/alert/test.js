import React from 'react';
import renderer from 'react-test-renderer';
import Alert from './';

const props = {
    error: new Error('Testing')
};

describe('Alert', () => {
    beforeEach(() => {
        // Fixes issue with Animated causing jest to hang
        // https://github.com/facebook/jest/issues/4359
        jest.useFakeTimers()
    });

    test('renders correctly with error prop', () => {
        const tree = renderer.create(<Alert {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });

    test('renders correctly without error prop', () => {
        const tree = renderer.create(<Alert/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
