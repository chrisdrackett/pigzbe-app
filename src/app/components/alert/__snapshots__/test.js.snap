// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renders correctly with error prop 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ff0000",
      "flex": 1,
      "flexDirection": "row",
      "height": 80,
      "justifyContent": "space-between",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ffffff",
        "fontFamily": "Poppins",
        "fontSize": 16,
        "marginBottom": 20,
        "marginLeft": 40,
        "marginRight": 20,
        "marginTop": 20,
        "textAlign": "left",
      }
    }
  >
    Testing
  </Text>
  <View
    style={
      Object {
        "marginRight": 20,
      }
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "alignSelf": "stretch",
              "color": "#003278",
              "fontFamily": "Poppins",
              "fontSize": 16,
              "fontWeight": "bold",
              "height": 45,
              "paddingLeft": 20,
              "paddingRight": 20,
              "paddingTop": 10,
              "textAlign": "center",
            },
            Object {
              "color": "#b3c1d6",
              "fontSize": 14,
              "textDecorationColor": "#ffffff",
              "textDecorationLine": "underline",
              "textDecorationStyle": "solid",
            },
          ]
        }
      >
        dismiss
      </Text>
    </View>
  </View>
</View>
`;

exports[`Alert renders correctly without error prop 1`] = `null`;
